src/Kdyby/Translation/Translator.php

```diff
@@ -143,7 +143,7 @@ public function translate($message, $count = NULL, $parameters = array(), $domai
 		$parameters = $tmp;
 
 		if ($count !== NULL && is_scalar($count)) {
-			return $this->transChoice($message, preg_replace('~[^0-9,.]~', '', $count), $parameters + array('%count%' => $count), $domain, $locale);
+			return $this->transChoice($message, preg_replace('~[^0-9,.\-]~', '', $count), $parameters + array('%count%' => $count), $domain, $locale);
```

Comment:
I see! I overlooked that. I don't really remember why was the preg_replace needed. It can probably be removed.